### PR TITLE
Base branch oc-4.4-kubernetes-1.17.0

### PR DESCRIPTION
**Pull request by sallyom**
/assign @soltysh

### CODE REVIEW

File: hack/make-rules/test.sh [hunk 1]
```diff
@@ -62,10 +62,24 @@ kube::test::find_dirs() {
...
       \) -name '*_test.go' -print0 | xargs -0n1 dirname | sed "s|^\./|${KUBE_GO_PACKAGE}/|" | LC_ALL=C sort -u
 
-    find ./staging -name '*_test.go' -not -path '*/test/integration/*' -prune -print0 | xargs -0n1 dirname | sed 's|^\./staging/src/|./vendor/|' | LC_ALL=C sort -u
+    find ./staging -not \( \
+        \( \
+          -path '*/test/integration/*' \
+          -o -path './staging/src/k8s.io/apiserver/pkg/server/options/*' \
+          -o -path './staging/src/k8s.io/apiserver/pkg/util/webhook/*' \
+          -o -path './staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/*' \
+          -o -path './staging/src/k8s.io/legacy-cloud-providers/gce/*' \
```
Comment (soltysh) [inline hack/make-rules/test.sh]: Add `-o -path './staging/src/k8s.io/csi-translation-lib/*' \ ` we don't care about that either.

Comment (sallyom) [reply hack/make-rules/test.sh]: ok done

**Pull request merged**